site/src/sphinx/server-decorator.rst

```diff
@@ -14,15 +14,16 @@ useful when `separating concerns`_.
 
 There are basically three ways to write a decorating service:
 
-- Implementing :api:`DecoratingServiceFunction`
-- Extending :api:`SimpleDecoratingService`
+- Implementing :api:`DecoratingHttpServiceFunction` and :api:`DecoratingRpcServiceFunction`
+- Extending :api:`SimpleDecoratingHttpService` and :api:`SimpleDecoratingRpcService`
 - Extending :api:`DecoratingService`
 
-Implementing ``DecoratingServiceFunction``
-------------------------------------------
+Implementing ``DecoratingHttpServiceFunction`` and ``DecoratingRpcServiceFunction``
+-----------------------------------------------------------------------------------
 
-:api:`DecoratingServiceFunction` is a functional interface that greatly simplifies the implementation of
-a decorating service. It enables you to write a decorating service with a single lambda expression:
+:api:`DecoratingHttpServiceFunction` and :api:`DecoratingRpcServiceFunction` is a functional interface that
```

Comment:
is a functional interface that greatly simplifies -> are functional interfaces that greatly simplify